buildlib/tools/zombie_killer.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -eEx
+
+# Search for zombie or defunct processes; kill them all
+out=$(ps -eo pid,stat | awk '$2 == "Z|defunct" {print $1}')
+if [ -z "$out" ]; then
+  echo "No zombies found"
+  exit 0
+else
+  for i in $out ; do
+    kill -9 "$i"
+  done
+fi
+
+# Search again; tell parents you killed their kids; kill parents
```

Comment:
Why do we need 2nd iteration?

Reply:
The original repo doesn't provide an explanation ([internal link](https://gitlab-master.nvidia.com/nbu-swx/hpc/swx-infrastructure/-/blob/master/scripts/azure/zombie_killer.sh)).
My guess is that if a process is still up after the first round, we need to approach its parents.